delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -0,0 +1,490 @@
+        option = [
+            opt.code
+            for opt in picking.postlogistics_option_ids
+            if opt.postlogistics_type == option
+        ]
+        assert len(option) <= 1
+        return option and option[0]
+
+    def _get_label_layout(self, picking):
+        label_layout = self._get_single_option(picking, "label_layout")
+        if not label_layout:
+            label_layout = picking.carrier_id.postlogistics_label_layout.code
+        return label_layout
+
+    def _get_output_format(self, picking):
+        output_format = self._get_single_option(picking, "output_format")
+        if not output_format:
+            output_format = picking.carrier_id.postlogistics_output_format.code
+        return output_format
+
+    def _get_image_resolution(self, picking):
+        resolution = self._get_single_option(picking, "resolution")
+        if not resolution:
+            resolution = picking.carrier_id.postlogistics_resolution.code
+        return resolution
+
+    def _get_license(self, picking):
+        """ Get the license
+
+        Take it from carrier and if not defined get the first license.
+
+        :return: license number
+        """
+        franking_license = picking.carrier_id.postlogistics_license_id
+        # if not franking_license:
+        #     company_licenses = picking.company_id.postlogistics_license_ids
+        #     if not company_licenses:
+        #         return None
+        #     franking_license = company_licenses[0]
+        return franking_license.number
+
+    def _prepare_attributes(self, picking, pack_num=None, pack_total=None):
+
+        services = [
+            code
+            for codes in (
+                option.code.split(",")
+                for option in picking.postlogistics_option_ids
+                if option.tmpl_option_id.postlogistics_type
+                in ("basic", "additional", "delivery")
+            )
+            for code in codes
+        ]
+        if not services:
+            raise exceptions.UserError(
+                _("Missing required delivery option on picking.")
+            )
+
+        attributes = {
+            "przl": services,
+            "weight": picking.shipping_weight,
+        }
+        option_codes = [option.code for option in picking.postlogistics_option_ids]
+        if "ZAW3217" in option_codes and picking.delivery_fixed_date:
+            attributes["deliveryDate"] = picking.delivery_fixed_date
+        if "ZAW3218" in option_codes and pack_num:
+            attributes.update(
+                {
+                    "parcelTotal": pack_total or picking.number_of_packages,
+                    "parcelNo": pack_num,
+                }
+            )
+        if "ZAW3219" in option_codes and picking.delivery_place:
+            attributes["deliveryPlace"] = picking.delivery_place
+        if picking.carrier_id.postlogistics_proclima_logo:
+            attributes["proClima"] = True
```

Comment:
Seems to be true by default. So I would always send this value

Reply:
I have fixed this, adding the case for False value